recipes/uvicore/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "uvicore" %}
+{% set version = "0.1.23" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/uvicore-{{ version }}.tar.gz
+  sha256: 1cb3ba1949dc5341ae04cf04379699e4b73afc49dbf17098ecd67aee200bb45b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry
```

```suggestion
    - poetry-core >=1.0.0
```
https://github.com/uvicore/framework/blob/9c21b85e9e470c6d789899340332a9abd0b26ab1/pyproject.toml#L123